### PR TITLE
Adds Juice Boxes!

### PR DESCRIPTION
**Pull request by ShibberinoShipperino**
New Juice Boxes Available to Drink!

Drinks:

drinks

Barry's Red Kola
Mingus Dew
Starman Sparkle
Queen's Haven
Aloha Coe
Blurred Limes
Mango Reinhardt

pls be gentle this is my first time attempting to code thank u i think this is still probably not gonna work or act like a can

Thanks to Danny (Counter) and Flipping Out for assistance.

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/drink_cold.dm [hunk 1]
```diff
@@ -158,6 +158,78 @@
...
+	id = "queenmilk"
+	description = "Looks like pretty good milk."
+	color = "##dfcda7"
+	adj_temp_cool = 3
+	drink_icon = "glass_white"
+	drink_name = "Glass of Queen's Haven"
+	drink_desc = "Looks like a glass full of milk, don't dip your cereal in here."
+
+
 /datum/reagent/consumable/drink/cold/rewriter/on_mob_life(mob/living/M)
```
Comment (Birdtalon) [inline code/modules/reagents/chemistry/reagents/drink_cold.dm]: Move this proc back up so it sits underneath `/datum/reagent/consumable/drink/cold/rewriter` it's not necessary to do so but it's better when someone looks at this file again to have the procs with the relevent objects.

**Review by Birdtalon — CHANGES_REQUESTED** [review]

**Comment by MrFroztee** [comment]
Are these available in a vendor, or are they just implemented into the code for further use?

**Comment by ShibberinoShipperino** [comment]
They aren't added to vendors yet, I've also updated it with the changes.

**Comment by msgerbs** [comment]
I would suggest making 3 sprites for each: A normal sprite, a sprite with a straw in it (for when it's opened) and a crushed sprite? I don't think there is currently a different sprite for open/closed but that could be added and it would be nice to have the sprites ready to go.

**Pull request merged**